salt/cache/redis_cache.py

```diff
@@ -7,6 +7,13 @@
 .. versionadded:: 2017.7.0
 .. versionchanged:: 3005
 
+To enable this cache plugin, the master will need the python client for redis installed.
+This can be easily installed with pip:
+
+.. code-block:: bash
+
+    pip install python-redis
```

Comment:
Looks like this is the official package: https://pypi.org/project/redis/#description

Needs to be changed from `python-redis` to `redis`

Reply:
since this is a small change I'll push the change

Reply:
I updated the docs to use the `pip.install` salt execution module since that module with handle the logic for installing pip packages for both classic and onedir packages.